frb_dart/lib/src/typed_data/html.dart

```diff
@@ -0,0 +1,160 @@
+@JS()
+library html_typed_data;
+
+import 'dart:collection';
+import 'package:js/js.dart';
+import 'package:fixnum/fixnum.dart';
+
+import 'dart:typed_data' hide Int64List, Uint64List;
+
+import 'package:js/js_util.dart';
+export 'dart:typed_data' hide Int64List, Uint64List;
+
+Int64List int64ListFrom(dynamic raw) => Int64List.from(raw);
+
+Uint64List uint64ListFrom(dynamic raw) => Uint64List.from(raw);
+
+// Borrowed from wasm_bindgen
+final _int64Shim = BigInt64Array(1);
+final _int32Shim = Int32List.view(_int64Shim.buffer);
+final _byteShim = Uint8List.view(_int64Shim.buffer);
+
+@JS('TypedArray')
+class TypedArray {
+  external ByteBuffer get buffer;
+  external int length;
+  external at(int index);
+}
+
+extension TypedArrayExt on TypedArray {
+  void operator []=(int index, value) {
+    setProperty(this, index, value);
+  }
+}
+
+@JS('BigInt64Array')
+class BigInt64Array extends TypedArray {
+  external factory BigInt64Array(Object lengthOrBuffer,
+      [int? offset, int? length]);
+
+  factory BigInt64Array.fromList(List<int> list) =>
+      BigInt64Array(list.map((n) => BigInt.from(n)).toList());
+
+  factory BigInt64Array.view(
+    ByteBuffer buffer, [
+    int offset = 0,
+    int? length,
+  ]) =>
+      BigInt64Array(buffer, offset, length);
+
+  factory BigInt64Array.sublistView(TypedData array,
+          [int offset = 0, int? length]) =>
+      BigInt64Array(array.buffer, offset, length);
+}
+
+@JS('BigUint64Array')
+class BigUint64Array extends TypedArray {
+  external factory BigUint64Array(Object lengthOrBuffer,
+      [int? offset, int? buffer]);
+
+  factory BigUint64Array.fromList(List<int> list) =>
+      BigUint64Array(list.map((n) => BigInt.from(n)).toList());
+
+  factory BigUint64Array.view(ByteBuffer buffer,
+          [int offset = 0, int? length]) =>
+      BigUint64Array(buffer, offset, length);
+
+  factory BigUint64Array.sublistView(TypedData array,
+          [int offset = 0, int? length]) =>
+      BigUint64Array(array.buffer, offset, length);
+}
+
+abstract class TypedList extends ListMixin {
+  TypedArray get inner;
+
+  /// How to cast a raw JS value to an acceptable Dart value.
+  dynamic _cast(Object? value);
+
+  /// How to convert a Dart integer-like value to an acceptable JS value.
+  dynamic _convert(Object? value);
+
+  @override
+  operator [](int index) => _cast(inner.at(index));
+
+  @override
+  void operator []=(int index, value) {
+    inner[index] = _convert(value);
+  }
+
+  @override
+  int get length => inner.length;
+
+  @override
+  set length(int newLength) => throw UnmodifiableTypedListException();
+}
+
+Int64 _castBigInt(Object bigInt) {
+  _int64Shim[0] = bigInt;
+  final lo = _int32Shim[0];
+  final hi = _int32Shim[1];
+  return Int64.fromInts(hi, lo);
+}
```

Comment:
Nice trick, so maybe we need to test it.

One approach is to add a unit test in frb_dart/test/..., and run it in web using `flutter test` (with target being web). Don't forget to setup CI :)

Reply:
This is already tested as part of the `Platform-specific support` test group, albeit transitively.